Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -352,31 +395,54 @@ function createWrapper(wrapperId) {
             </td>
 
           </sec:authorize>
-          <td align="justify"><font size="2"> <a
+          <td align="justify"> <a
               id="${concept.entry.id}" data-toggle="modal"
               data-target="#detailsModal"
               data-conceptid="${concept.entry.id}"><c:out
-                  value="${concept.entry.word}"></c:out></a></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.id}"></c:out></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.wordnetId}"></c:out></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.pos}"></c:out></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.conceptList}"></c:out></font></td>
+                  value="${concept.entry.word}"></c:out></a></td>
+          <td class="entryID" align="justify" id="entryId" ><c:out
+                value="${concept.entry.id}"></c:out></td>
+          <td align="justify"><c:out
+                value="${concept.entry.wordnetId}"></c:out></td>
+          <td align="justify"><c:out
+                value="${concept.entry.pos}"></c:out></td>
+          <td align="justify"><c:out
+                value="${concept.entry.conceptList}"></c:out></td>
           <td align="justify">
                 <div class="scrollable" style="max-height: 100px; max-width: 400px;">
                     <c:out value="${concept.description}" escapeXml="false"></c:out>
                 </div>
           </td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.type.typeName}"></c:out></font></td>
+          <td align="justify"><c:out
+                value="${concept.type.typeName}"></c:out></td>
+                
+         <!-- Enabling Disabling the Review button -->  
+          <c:choose>
+ 		 	<c:when test="${concept.reviewRequest.comment == null}"> <!-- Testing if the comment has already been provided. -->
+ 		 	   	<td align="center"><button data-concept-id="${concept.entry.id}" type="button" style="color:white; background:#FF9B22;margin-bottom: 15px;" class="btnReview" data-toggle="modal" data-target="#myModal">Review</button></td>
+  			</c:when>
+  			<c:otherwise>
+   			    <td onclick="showForm('${concept.reviewRequest.comment}');" align="center"><div class="fa fa-envelope"></div></td>
+ 			</c:otherwise>
+		  </c:choose>
         </tr>
       </c:forEach>
     </tbody>
   </table>
 
+  
+<style> div.floatingform { position: absolute; right: 220px; top:220px; width:5%; margin:0 auto; height: auto;} </style>
```

Comment:
if you need some page specific css, let's add that as style info the element directly